### PR TITLE
Update 'docs/CONTRIBUTING.md'

### PR DESCRIPTION
**Pull request by Hedreon**
To replace references of 'master' to 'Main' and to fix grammar issues.

### CODE REVIEW

**Comment by Hedreon** [comment]
If any problems, comment in the PR.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ There are other branches as well on Quick Pad, these are usually for new feature
 
 ## Contributing to Quick Pad
 
-To get started, create a fork of the repository on GitHub. You will need to do that to make any changes since the Quick Pad repo is restricted to a few contributors of the app. If you decide to make a change you should always base it off  the `Main` branch. Once your change is completed you can open a pull request which we will review and if we like it we will merge it in. 😀
+To get started, create a fork of the repository on GitHub. You will need to do that to make any changes since the Quick Pad repo is restricted to a few contributors of the app. If you decide to make a change you should always base it off  the `Main` branch. Once your change is completed, you can open a pull request in which we will review your pull request and if we like it, we will merge it in. 😀
```
Comment (yaichenbaum) [inline docs/CONTRIBUTING.md:15]:
```suggestion
To get started, create a fork of the repository on GitHub. You will need to do that to make any changes. If you decide to make a change you should always base it off the `Main` branch. Once your change is completed, you can open a pull request in which we will review and merge if approved.
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Comment by Hedreon** [comment]
Suggestion merged.

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,11 +8,11 @@
...
+There are other branches as well on Quick Pad, these are usually for new features or changes that are still a work in progress and not yet ready to be merged into the `Main` branch. These branches are temporary and will be removed once the changes are merged into the `Main` branch.
 
 ## Contributing to Quick Pad
 
-To get started, create a fork of the repository on GitHub. You will need to do that to make any changes since the Quick Pad repo is restricted to a few contributors of the app. If you decide to make a change you should always base it off  the `master` branch. Once your change is completed you can open a pull request which we will review and if we like it we will merge it in 😀
+To get started, create a fork of the repository on GitHub. You will need to do that to make any changes. If you decide to make a change you should always base it off the `Main` branch. Once your change is completed, you can open a pull request in which we will review and merge if approved.
```
Comment (yaichenbaum) [inline docs/CONTRIBUTING.md:15]:
```suggestion
To get started, create a fork of the repository on GitHub. You will need to do that to make any changes. If you decide to make a change you should always base it off the `Main` branch. Once your change is completed, you can open a pull request which we will review and merge if approved.
```

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
Thank you!

**Pull request merged**